test/jdk/javax/swing/JMenuItem/JMenuItemSetAcceleratorTest.java

```diff
@@ -89,7 +104,8 @@ public static void main(String[] args) throws Exception {
                     "Hasn't received the JMenuItem action event by pressing "
                         + "accelerator keys, test fails.");
             }
-            System.out.println("Test passed, received action event on menu item.");
+            System.out
+                .println("Test passed, received action event on menu item.");
```

Comment:
My IDE’s formatter auto wraps this line for 80 chars.
After every code change & formatting, I have to manually unwrap these lines.
There are 3 lines to  unwrap manually.

Reply:
> My IDE’s formatter auto wraps this line for 80 chars. After every code change & formatting, I have to manually unwrap these lines. There are 3 lines to unwrap manually.

Then disable automatic reformatting of the entire file. It won't do much good to you in the long term anyway because it may modify the lines you didn't touch and eventually introduce unnecessary noise into the changes. This is what we're seeing now.